src/telliot_feed_examples/sources/price/spot/bittrex.py

```diff
@@ -47,7 +47,11 @@ async def get_price(self, asset: str, currency: str) -> OptionalDataPoint[float]
         d = self.get_url(request_url)
 
         if "error" in d:
-            logger.error(d)
+            if "JSON Decode Error" in d["error"]:
+                error_msg = d["error"] + " response from bittrex api"
```

Comment:
This needs to be more specific to the bittrex API, as there could be other JSON decoding errors that we'll miss with this.
- Check in the html/text of the response for something about bittrex limiting usage and
- Make the log message say that bittrex is rate limiting